src/shared/bg_pmove.cpp

```diff
@@ -2125,13 +2124,13 @@ PM_CheckLadder
 Check to see if the player is on a ladder or not
 =============
 */
-static void PM_CheckLadder()
+static void PM_CheckLadder( const pmove_t * pm_, pml_t & pml )
```

Comment:
`pm` should be global either everywhere or nowhere.

Reply:
I think I fixed it, but have a doubt.